deploy/components/example_cluster.yaml

```diff
@@ -73,6 +73,7 @@ spec:
             requests:
               cpu: 1000m
               memory: 512Mi
+              ephemeral-storage: 1Gi
```

Comment:
hmmm do we need to add ephemeral storage/is there a benefit to it? If not, can we revert the changes to this file for the sake of keeping the PR minimal?

Reply:
True unrelated (I need it to deploy on my cluster but I can do it as a separate PR).